user/downloading-installing-upgrading/installation-guide.md

```diff
@@ -133,12 +133,30 @@ settings.
 ### Getting to the boot screen
 
 Just after you power on your machine, make the Qubes OS medium available to the
-computer by inserting your DVD or USB drive. Shortly after the Power-on
-self-test (POST) is completed, you should be greeted with the Qubes OS boot
-screen. 
+computer by inserting your USB drive.
+
+Then you'll need to power on the computer and immediately press a specific key to
+enter what's called the "BIOS" of you computer. The key to press varies wildly from
+one computer to another. `Esc`, `Del` and `F10` are
+common keys. If you're not sure about the key to press you'll have to view in the
+machine's user guide or serch the web for `accessing BIOS <COMPUTER_MODEL>` 
+(replacing with your specific computer model).
+
+The BIOS menu will looks something like the following pictures:
+
+| ![](/attachment/doc/Thinkpad-t430-bios-main.jpg)  | |
+|----|----|
+| [CC-BY-SA-4.0 Vitaly Zdanevich](https://commons.wikimedia.org/wiki/File:Thinkpad-t430-bios-main.jpg) | ADD here UEFI
+
+Within the BIOS you'll navigate to the boot menu and either select to boot from your USB device or order the boot devices in such a way that the USB stick is the first one. Then save the configurations and restart your computer. After this the computer should show the Qubes boot screen:
 
 ![Boot screen](/attachment/doc/boot-screen.png)
 
+<div class="alert alert-info" role="alert">
+  <i class="fa fa-info-circle"></i>
+  <b>Note:</b> Please note that there may be other thing you need to do in BIOS. *You may need to change them later* if the installer still refuses to start (deactivating secure boot can solve it) or if when installing Qubes shows an error saying you computer does not support virtualizaiton technology (where you'll need to activate virtualization technology).
```

Comment:
These asterisks will be literal asterisks, not italics, since you're currently inside of an alert div. You'll have to use `<em>` here if you want emphasis, but I don't think emphasis makes anything clearer in this case.

Note that there are some typos and unclear language here. I can do a copyediting pass to fix these before merging, but in case you're curious what I mean, here are some examples:
- "other thing" -> "other things"
- "in BIOS" -> "in the BIOS"
- "You may need to change them later" -- Change what?
- "deactivating secure boot can solve it" -- I don't think this will solve ever case of the installer refusing to start.
- "if when installing Qubes shows an error" -- I read this as "if, when installing Qubes, shows an error" which makes no sense, but I'm guessing you meant "if, when installing, Qubes shows an error." IMO, punctuation makes a big difference here. But it's also a bit lax to say "Qubes" shows an error. More precise would be "the installer," since Qubes OS isn't even installed yet.

Reply:
I'll revise this. The story is the same. I started working on this and ended up not having enough time. I really wish there was a personal draft thing. Would have saved you a bit of time revising this lower than acceptable WIP. Sorry

Reply:
> I'll revise this. The story is the same. I started working on this and ended up not having enough time. I really wish there was a personal draft thing. Would have saved you a bit of time revising this lower than acceptable WIP. Sorry

Oh, no worries! There actually is a way to make a draft PR. I'm not sure if it requires special permission or something, but it shouldn't:

https://github.blog/2019-02-14-introducing-draft-pull-requests/

Reply:
Yes, I could have made those. But I believe that's just a costmetic thing (very much like "[WIP]" in the title). In realility is is publicly listed just like all other PRs.

Reply:
> Yes, I could have made those. But I believe that's just a costmetic thing (very much like "[WIP]" in the title). In realility is is publicly listed just like all other PRs.

Ah, I actually didn't even realize, until this very moment, that the `[WIP]` in your PR's title meant you didn't want it reviewed yet, lol. Sorry about that. After seeing people include all sorts of `[tags]` in the titles of their issues and PRs over the years (many of them weird and pointless, unlike yours), my brain just filters them out now.

Submitting the PR as a draft, although public, is much better, because I can't merge it while it's a draft, so it makes it absolutely clear what the intention is.